.github/workflows/CI.yml

```
@@ -0,0 +1,97 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - '*'
```

`*` doesn't include branches with slashes, which is a terrible decision if you ask me. So I try to basically always use `**` for branches.

```suggestion
      - **
```